dex/src/test/scala/com/wavesplatform/dex/market/OrderBookActorSpecification.scala

```diff
@@ -109,6 +109,13 @@ class OrderBookActorSpecification
       tp.expectMsg(OrderBookRecovered(pair, Some(50)))
     }
 
+    "recovery - if there is a matching rule - DEX-775" in obcTestWithPrepare(
```

Comment:
DEX-755 in test name :expressionless: 
But in the other hand its ok for a link to the problem description